### PR TITLE
Upgrade to docs to hubStyle template and conform pkg to hubverse standards

### LINKED ISSUE #17: Upgrade Docs to hubStyle
## Update package config

- [x] Create `enhancement/hubstyle` branch
- [x] Run `hubDevs::use_hubdev_pkgdown(add_logo = TRUE)` to update pkgdown
- [x] Add logo to your README
- [x] Append standard footer to README with `hubDevs::append_hubdev_readme_footer()`. Render
- [x] Run `hubDevs::use_hubdev_community()` to update community docs
- [x] Run `hubDevs:::hubdev_ignore()` to ignore std files
- [x] Check authors
- [x] Add `hubverse` & `r-package` topics to repos

## Set up Netlify PR Previews

- [x] Log into Hubverse Netlify account and drag in pkg docs direcory to creat new site.
- [x] Edit site name with `[pkg_name]-pr-previews` and copy `NETLIFY_SITE_ID`
- [x] _Create a `NETLIFY_AUTH_TOKEN` if required._
- [x] Add `NETLIFY_AUTH_TOKEN` & `NETLIFY_SITE_ID` to repository secrets

### Push to Github, open a PR and check Preview 🎉

### PR DESCRIPTION
**Pull request by annakrystalli**
Resolves #17

### CODE REVIEW

**Review by nickreich — CHANGES_REQUESTED** [review]
seems like we should fix the lintr check warnings before merging? Otherwise it looks good to me.

**Comment by annakrystalli** [comment]
Sure. I was going to let you have a look at the code unless you want me to fix the linting issues? The issues are in the `data-raw` files.

**Comment by annakrystalli** [comment]
Fixed the linting issues and removed a library call to `tidyverse` in favour of `dplyr`. Ready for your final review whenever you are @nickreich

File: .github/workflows/pkgdown.yaml
Comment (nickreich) [inline .github/workflows/pkgdown.yaml:1]: I don't see where the site is rebuilt for non-PR workflows if this file is removed. The workflows/pgkdown-netlify-preview.yaml file looks like it only handles PR-related workflows, although I'm likely missing something about the way it is set up.

File: .github/workflows/pkgdown-netlify-preview.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        with:
+          extra-packages: any::pkgdown, local::.
+          needs: website
+
+      - name: Build site
+        run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
+        shell: Rscript {0}
+
+      - name: Deploy production to GitHub pages 🚀
+        if: contains(env.isPush, 'true')
```
Comment (nickreich) [inline .github/workflows/pkgdown-netlify-preview.yaml:50]: I see - maybe this is the step where it's saying "if this is a push to the main branch then deploy to GitHub pages"?

**Review by nickreich — APPROVED** [review]
Thanks @annakrystalli for making the linting changes. I've been at a meeting and haven't had a chance to get to it. The changes look good. And I think I convinced myself that the pkgdown-netlify-preview workflow has appropriate switches for when there is a PR vs a push.

File: .github/workflows/pkgdown-netlify-preview.yaml (hunk 1 above)
Comment (annakrystalli) [reply .github/workflows/pkgdown-netlify-preview.yaml:50]: Right! That's the one

**Pull request merged**